slides/05_collaboration.qmd

```diff
@@ -319,23 +319,23 @@ gitGraph
 sequenceDiagram
```

Comment:
in the presentation html the text could not be read clearly
![Uploading image.png…]()

Reply:
@clarkliming How you generated that? It looks good in my output
![image](https://user-images.githubusercontent.com/48791914/226793530-77115410-bd04-4459-8652-0cda539ba3ef.png)

Reply:
probably this is a duplicated message as I said the same here https://github.com/openpharma/workshop-r-swe/pull/10#discussion_r1144185343